app/src/config/env.ts

```diff
@@ -0,0 +1,13 @@
+export interface EnvVars {
+    LAUNCHDARKLY_API_KEY: string;
+    LAUNCHDARKLY_APP_NAME: string;
+}
+
+const envVars: EnvVars = {
+    LAUNCHDARKLY_API_KEY: process.env.LAUNCHDARKLY_API_KEY || '',
```

Comment:
These have been set for the appId using some forge cli commands. For more info see https://developer.atlassian.com/platform/forge/environments-and-versions/